### ISSUE TITLE
Cisco has renamed vlans 1003 and 1005 causing Macsuck delays and SNMP Authentication errors on newer Cisco Switches

### ISSUE THREAD

**Issue by ednpnoble** [netdisco/netdisco#1037]
In newer versions of IOS, Cisco has renamed pre-built VLANs 1003 and 1005 from their respective names of token-ring-default and trnet-default to trcrf-default and trbrf-default.

Per documentation at https://github.com/netdisco/netdisco/wiki/Configuration#macsuck_no_vlan these new names are not ignored by default.

> Default: fddi-default, token-ring-default,fddinet-default,trnet-default.

## Current Behavior
Macsuck stalls on vlans 1003 and 1005 instead of skipping
```
[1886867] 2023-05-15 23:50:31 debug  [10.1.1.1] macsuck - VLANs: 1,1002,1003,1004,1005,2100,2101
[1886867] 2023-05-15 23:50:31 debug  [10.1.1.1] macsuck VLAN 1002 - skipped by macsuck_no_vlan config
[1886867] 2023-05-15 23:50:31 debug  [10.1.1.1] macsuck VLAN 1004 - skipped by macsuck_no_vlan config
[1886867] 2023-05-15 23:50:31 debug [10.1.1.1] reindexing to vlan 1 (ver: 2, class: SNMP::Info::Layer3::CiscoSwitch)
[1886867] 2023-05-15 23:50:31 debug [10.1.1.1] reindexing to vlan 1003 (ver: 2, class: SNMP::Info::Layer3::CiscoSwitch)
[1886867] 2023-05-15 23:50:58 debug [10.1.1.1] reindexing to vlan 1005 (ver: 2, class: SNMP::Info::Layer3::CiscoSwitch)
[1886867] 2023-05-15 23:51:25 debug [10.1.1.1] reindexing to vlan 2100 (ver: 2, class: SNMP::Info::Layer3::CiscoSwitch)
```

Newer Cisco Switch running IOS-XE 17.6 code
```
Cisco9300#show vlan id 1001-1005

VLAN Name                             Status    Ports
---- -------------------------------- --------- -------------------------------
1002 fddi-default                     act/unsup
1003 trcrf-default                    act/unsup
1004 fddinet-default                  act/unsup
1005 trbrf-default                    act/unsup
```

Older Cisco Switch running IOS 12.2
```
OlderCiscoSwitch#show vlan id 1001-1005

VLAN Name                             Status    Ports
---- -------------------------------- --------- -------------------------------
1002 fddi-default                     act/unsup
1003 token-ring-default               act/unsup
1004 fddinet-default                  act/unsup
1005 trnet-default                    act/unsup

```

## Possible Solution
Suggest adding the names trcrf-default and trbrf-default to the default list of VLAN names that are ignored for macsuck in addition to the existing list.  Also recommend updating the documentation listed above to reflect the potential change.

Existing Configuration
https://github.com/netdisco/netdisco/blob/3f8ffe787f3af1b8eacf1e712475b1096c490b2e/share/config.yml#L361-L369

Add the following lines:
```  
  - 'trcrf-default'
  - 'trbrf-default'
```

## Steps to Reproduce (for bugs)

1. netdisco-do macsuck -D -d 10.1.1.1  ( A newer switch with the new VLAN Names like Catalyst 9300 running IOS-XE 17.6)
2. Observe behavior of NetDisco when it attempts to macksuck vlans 1003 and 1005.  Expected behavior is to skip these vlans like 1002, but is not (because the name isn't being recognized)

## Context
This issue has two noted impacts:
1. Macsuck takes longer to run because it fails on these vlans 
2. 
## Your Environment

* Netdisco version used: 2.52.8
* SNMP::Info version used: 3.84

## Config info (deployment.yml)
Default configuration, however, I am mitigating the behavior manually in my config with the following (Not recommended for regular production use, we should use the VLAN names to avoid compatibility with other platforms.)
```
macsuck_no_vlan:
 - 1002
 - 1003
 - 1004
 - 1005
```

**Comment by ednpnoble** [comment]
Description of recommended fix (Updating original text to reflect this)

https://github.com/netdisco/netdisco/blob/3f8ffe787f3af1b8eacf1e712475b1096c490b2e/share/config.yml#L361-L369

Add the following lines:
```  
  - 'trcrf-default'
  - 'trbrf-default'
```

**Comment by ollyg** [comment]
Many thanks for the detailed report @ednpnoble !

**Comment by ollyg** [comment]
thanks again, patched in 01928fa8 and will be in next release
